tests/phpunit/includes/testcase-xml.php

```diff
@@ -41,7 +41,7 @@ public function loadXML( $xml, $options = 0 ) {
 	 */
 	public function normalizeXML( $xml, $options = 0 ) {
 		if ( ! class_exists( 'XSLTProcessor' ) ) {
-			$this->markTestSkipped( 'This test requires the XSL extension.' );
+			$this->fail( 'This test requires the XSL extension.' );
```

Comment:
Why should the tests fail when the extension is missing ? A contributor could be running the tests locally and not have the extension enabled. Marking a test as skipped in such a case is the appropriate way to handle this. Not failing the test suite.